File: app/src/main/java/com/marked/pixsee/frienddetail/viewholders/TypingHolder.java
package com.marked.pixsee.frienddetail.viewholders;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import com.marked.pixsee.R;
import com.zl.reik.dilatingdotsprogressbar.DilatingDotsProgressBar;

/**
 * Created by Tudor Pop on 02-Jan-16.
 * This makes the animation dots happens
 */
public class TypingHolder extends RecyclerView.ViewHolder {
	public TypingHolder(View itemView) {
		super(itemView);
	}
	public void start(){
		((DilatingDotsProgressBar)itemView.findViewById(R.id.dot)).showNow();
	}
	public void stop(){
		((DilatingDotsProgressBar)itemView.findViewById(R.id.dot)).hideNow();
	}
}

File: app/src/main/java/com/marked/vifo/ui/adapter/viewholders/TypingHolder.kt
package com.marked.vifo.ui.adapter.viewholders

import android.support.v7.widget.RecyclerView
import android.view.View
import kotlinx.android.synthetic.main.item_typing_message.view.*

/**
 * Created by Tudor Pop on 02-Jan-16.
 * This makes the animation dots happens
 */
class TypingHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
//	private var dot = itemView.findViewById(R.id.dot) as DilatingDotsProgressBar

	fun start() {
		itemView.dot.showNow()
//		dot.mBaseDot.setAnimationStatus(BaseDot.AnimStatus.START)
    }

	fun stop() {
		itemView.dot.hideNow()
//		dot.mBaseDot.setAnimationStatus(BaseDot.AnimStatus.CANCEL)
	}
}
